### ISSUE TITLE
Ensure all files have CODEOWNERS

### ISSUE THREAD

**Issue by dhruvkb** [WordPress/openverse#1207]
## Problem

Considering how the monorepo is a complex mix of projects, languages and stacks, we cannot pick from the pool of all maintainers to review PRs. Hence the CODEOWNERS file is a complex Venn diagram of teams related to each part of the codebase.

In this complexity, it is easy for files and directories to be left with no owner at all.

## Description

https://github.com/beaugunderson/codeowners is a tool to interact with the `CODEOWNERS` file. It can be used to find files that have no owner. Running it as a script in pre-commit can fix this issue.

Integrating it with pre-commit can help prevent these situations from being committed at all, but that is blocked on upstream issue https://github.com/beaugunderson/codeowners/issues/36 to be resolved.

**Comment by dhruvkb** [comment]
Closing as a duplicate of #3554 which has a better solution and a contributor working on it.
